pkgs/applications/science/electronics/kicad/default.nix

```diff
@@ -59,7 +172,7 @@ stdenv.mkDerivation rec {
 
   nativeBuildInputs = [ makeWrapper ]
     ++ optionals (scriptingSupport)
-      [ python.pkgs.wrapPython ];
+    [ python.pkgs.wrapPython ];
 
   # wrapGAppsHook added the equivalent to ${base}/share
   # though i noticed no difference without it
```

Comment:
Nitpick:

```suggestion
  # We are emulating wrapGAppsHook, along with other variables to the wrapper
```

Reply:
i wasn't emulating wrapGAppsHook, since that misbehaves